### ISSUE TITLE
Box2dLights not rendering

### ISSUE THREAD

**Issue by adam-law** [Spasi/LWJGL-FX#9]
I know it might seem crazy to you that we're continuing to use this for production, but it's been working well, so far. Well, up until today.

I've been banging my head for a couple of days wondering why the lights from Box2dlights won't show, checked everything, then thought of running stuff as a LWJGLApplication, and lo, "let there be light...and there was light."

Care to share your thoughts?

**Comment by Spasi** [comment]
Do you see anything if you render something that doesn't depend on box2dlights?

**Comment by adam-law** [comment]
Yes, everything else shows, even the debug borders of Box2D. My guess is either Box2dLights' FBO usage, or shaders. I haven't gotten to the part of using either on my work, apart from Box2dLights, of course.

Edit: continuing work, I found the RayHandler for Box2dLights, when doing its update, also interferes with the rendering of 3D objects through ModelBatch. Without the rayhandler update, 3d objects display properly. Don't know if that's helpful info or not.
